app/src/main/java/com/quran/labs/androidquran/ui/util/TranslationsSpinnerAdapter.java

```diff
@@ -129,4 +143,19 @@ public void updateItems(String[] translationNames,
   public interface OnSelectionChangedListener {
     void onSelectionChanged(Set<String> selectedItems);
   }
+
+  public void setOnClickListener(View.OnClickListener onClickListener) {
+    this.textListener = onClickListener;
+  }
+
+  public String[] updateTranslationNames(String[] translationNames) {
+    List<String> translationsList = new ArrayList<>();
+    for (String translation : translationNames) {
```

Comment:
jzk, I made the change to a loop based on @ahmedre earlier comment "i'd then replace it with a loop - i.e.

List<String> transList = new ArrayList<>();
for (String translation : translationNames) {
   transList.add(translation);
}" , Please let me know if the replacement is what we want as a final state. thanks